sensio/sphinx/codeblock.py

```diff
@@ -0,0 +1,16 @@
+from sphinx.directives.code import CodeBlock
+
+"""
+The only change with the built-in CodeBlock class that comes with Sphinx
+is that linenos is always on.
+"""
+class NumberedCodeBlock(CodeBlock):
+    def run(self):
+        literal = super(NumberedCodeBlock, self).run()[0];
+        literal['linenos'] = 'linenos'
```

Comment:
No, but linenos is not correct as well:

 > `linenos`
 >If set to ``'table'``, output line numbers as a table with two cells,
 >one containing the line numbers, the other the whole code.  This is
 >copy-and-paste-friendly, but may cause alignment problems with some
 >browsers or fonts.  If set to ``'inline'``, the line numbers will be
 >integrated in the ``<pre>`` tag that contains the code (that setting
 >is *new in Pygments 0.8*).

 >For compatibility with Pygments 0.7 and earlier, every true value
 >except ``'inline'`` means the same as ``'table'`` (in particular, that
 >means also ``True``).

 >The default value is ``False``, which means no line numbers at all.

I'll change it to table, as that seems what is used by symfony.com.